### PR TITLE
added new payload keys

### PR DESCRIPTION
**Pull request by Apurer**
Based on [Apple Documentation Generating a Remote Notification](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification).

### CODE REVIEW

**Comment by Apurer** [comment]
Based on [Apple Documentation Generating a Remote Notification](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification)

File: notification.go [hunk 1]
```diff
@@ -30,24 +30,33 @@ type APS struct {
...
-	// ThreadID presents the app-specific identifier for grouping notifications.
-	ThreadID string `json:"thread-id,omitempty"`
+	// Content available apps launched in the background or resumed.
+	ContentAvailable *int `json:"content-available,omitempty"`
+
+	// The notification service app extension flag. If the value is 1, the system passes the notification to your notification service app extension before delivery.
+	MutableContent *int `json:"mutable-content,omitempty"`
+
+	// The identifier of the window brought forward. The value of this key will be populated on the UNNotificationContent object created from the push payload.
+	TargetContentId string `json:"target-content-id,omitempty"`
```
Comment (edganiukov) [inline notification.go]:
```suggestion
	TargetContentID string `json:"target-content-id,omitempty"`
```

**Review by edganiukov — CHANGES_REQUESTED** [review]
Thanks for the PR, just a small comment.

File: notification.go (hunk 1 above)
Comment (Apurer) [reply notification.go]: Sure :)

**Review by edganiukov — APPROVED** [review]

**Pull request merged**